flink-python/src/main/java/org/apache/flink/table/runtime/operators/python/AbstractStatelessFunctionOperator.java

```diff
@@ -53,17 +51,11 @@
     /** The input logical type. */
     protected final RowType inputType;
 
-    /** The output logical type. */
-    protected final RowType outputType;
-
-    /** The offsets of user-defined function inputs. */
-    protected final int[] userDefinedFunctionInputOffsets;
-
     /** The user-defined function input logical type. */
-    protected transient RowType userDefinedFunctionInputType;
+    protected final RowType userDefinedFunctionInputType;
 
     /** The user-defined function output logical type. */
-    protected transient RowType userDefinedFunctionOutputType;
+    protected final RowType userDefinedFunctionOutputType;
```

Comment:
very nit: can we shorten the names, it is difficult to read at various locations
```
userDefinedFunctionOutputType -> udfOutputType
userDefinedFunctionInputType -> udfInputType
udafInputGeneratedProjection -> udafInputProjection
groupKeyGeneratedProjection -> groupKeyProjection
userDefinedFunctionInputOffsets -> udfInputOffsets
```

Reply:
I prefer to keep `GeneratedProjection` as it clashes with `Projection` otherwise, for the others i will change them